### PR TITLE
Fix Gousto (issue #376) by using schema

### PR DESCRIPTION
**Pull request by arbrennan**
Related to issue #376 

Updated the reference HTML to match live site, revert scraper to use schema.
Updated tests.

My first PR so please let me know if I need to update anything. Also please check the canonical URL test - the schema returns `https://test.example.com/` I'm not sure what to do about this

### CODE REVIEW

File: tests/test_gousto.py [hunk 1]
```diff
@@ -25,27 +25,28 @@ def test_yields(self):
...
-                "1 pot of double cream (227ml) †",
-                "1/2 beef stock cube †",
-                "200g linguine †",
-                "Olive oil",
-                "pepper",
-                "salt",
+                "1 pot of double cream (227ml)",
+                "1/2 beef stock cube",
+                "200g linguine"
+                # "Olive oil",
```
Comment (bfcarpio) [inline tests/test_gousto.py:40]: Is there a reason that these are commented out as opposed to being removed entirely?

File: tests/test_gousto.py [hunk 2]
```diff
@@ -10,9 +10,9 @@ def test_host(self):
         self.assertEqual("gousto.co.uk", self.harvester_class.host())
 
     def test_canonical_url(self):
+        print(self.harvester_class.canonical_url())
```
Comment (bfcarpio) [inline tests/test_gousto.py:13]: Please remove print statements from tests.

**Review by bfcarpio — CHANGES_REQUESTED** [review]
Thank you for the fixes. Just some minor commentary and if it passes tests I'd be happy to merge it in.

**Comment by hhursev** [comment]
Merging 🍺  I'll fix the print statements after the merge.

**Pull request merged**